metadata.display.yaml

```diff
@@ -48,6 +49,20 @@ spec:
         tables:
           name: tables
           title: Tables
+          level: 1
+          altDefaults:
+            - type: ALTERNATE_TYPE_DC
+              value:
+                table1:
+                  table_name: table-1
         zones:
           name: zones
           title: Zones
+          level: 1
+          altDefaults:
```

Comment:
@amrkk123 You can remove alt default for this as a default is being added in variables.tf in another PR https://github.com/GoogleCloudPlatform/terraform-google-bigtable/pull/20/files#diff-05b5a57c136b6ff596500bcbfdcff145ef6cddea2a0e86d184d9daa9a65a288eR68